src/types/UiaTextRangeBase.cpp

```diff
@@ -201,11 +127,11 @@ void UiaTextRangeBase::Initialize(_In_ const UiaPoint point)
         _TranslatePointFromScreen(&clientPoint);
 
         const COORD currentFontSize = _getScreenFontSize();
-        row = (clientPoint.y / currentFontSize.Y) + viewport.Top;
+        row = gsl::narrow<SHORT>(clientPoint.y / static_cast<LONG>(currentFontSize.Y)) + viewport.Top;

```

Comment:
This feels like it could make use of some of the new Chromium math to either report the failure to fit into a SHORT to you or to clamp it to the maximum value of a short.